SFDigitalServices/sf-dahlia-lap
# Fix missing checkboxes due to pattern library issue DAH-657

DAH-657

To test:
Load e.g. the applications table and make sure that the checkboxes show checkmarks instead of just a blue box
![image](https://user-images.githubusercontent.com/11825994/104375738-b7e77300-54d8-11eb-89e3-4250e78cf838.png)

Pattern lib commit: https://github.com/SFDigitalServices/sf-dahlia-pattern-library/commit/ad1dd02301fc9c2f74d5e1d77fd825f70406584c
